pkg/client/mapiserver/volumes_list.go

```diff
@@ -13,7 +13,7 @@ const (
 // ListVolumes and return them as obj
 func ListVolumes(obj interface{}) error {
 
-	body, err := getRequest(GetURL()+listVolumePath, "", false)
+	body, err := getRequest(GetURL()+listVolumePath, "", true)
```

Comment:
what does `true` or `false` here mean?

Reply:
Its for getting the response from the http response body in case of any error...basically non 200 response

Reply:
if true it will read the response from body ...